app/tests/helpers/check_helpers.py

```diff
@@ -0,0 +1,41 @@
+from unittest.mock import patch
```

Comment:
`pytest` already has `monkeypatch`

Reply:
will check

Reply:
wont be using monkeypatch 🧛

Reply:
Why?

Reply:
Dont wanna refactor :v, and also it have less feature :v